components/openindiana/time-slider/Makefile

```diff
@@ -21,7 +21,7 @@ COMPONENT_VERSION=	0.2.101
 COMPONENT_LICENSE=	CDDL
 COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
 
-COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count)
+COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count).1
```

Comment:
It is just to uprev the package version/revision so it is installed on the next update.  Strictly speaking it is not needed, but we would need to wait for the next push to the time-slider repo.

The other option to force the package version update is to make a fake push to the time-slider repo.

Or, should I just update the COMPONENT_VERSION to 0.2.102 or 0.2.101.1?